### DISCUSSION TITLE
Isaac Sim Orbit - Windows 11 Native

### DISCUSSION THREAD

**Discussion by Aronnaxx** [isaac-sim/IsaacLab#258] [Q&A] [answered]
Hi all,

I've been trying to get Orbit to work via Isaac Sim in Windows natively, however I've hit a lot of roadblocks with things like cMake and other items. 

I notice that in the Isaac Sim Extensions, Orbit is included under '3rd party' - is there a way to be able to utilize it in Windows via Isaac Sim's GUI to perform the same items that it would via Linux? I've gotten it to work as-expected on Linux (save for Robomimic, which seems to be a different issue), but Windows still escapes me.

**Comment by Mayankm96** [comment, marked as answer]
Hi, this is now supported. Please re-open the discussion if the problem remains unsolved.
